_layouts/default.html

```diff
@@ -9,6 +9,14 @@
         <!-- Javascript files -->
         <script src="/js/jquery.min.js"></script>
         <script src="/js/bootstrap.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.14.7/react.min.js"
```

Comment:
This is going to add react to pages that possibly do not need it right? Could we add another layout for this? Or are we going to use react more?